webextensions/api/privacy.json

```diff
@@ -161,7 +161,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": true
+                  "version_added": "18"
```

Comment:
This is based on the actual version when the privacy API was moved out of the experimental status.